Pod/Classes/TUSResumableUpload.m

```diff
@@ -14,8 +14,11 @@
 #define HTTP_PATCH @"PATCH"
 #define HTTP_POST @"POST"
 #define HTTP_HEAD @"HEAD"
-#define HTTP_OFFSET @"Offset"
-#define HTTP_FINAL_LENGTH @"Final-Length"
+#define HTTP_OFFSET @"Upload-Offset"
+#define HTTP_FINAL_LENGTH @"Upload-Length"
```

Comment:
It would be nice to have the constant renamed to HTTP_UPLOAD_LENGTH to match the header's name.